packages/vue/src/useSignUpEmailPassword.ts

```diff
@@ -1,4 +1,4 @@
-import { computed, ToRefs, unref } from 'vue'
```

Comment:
What do you think about enabling this in VSCode settings on a repository level through `.vscode`? I found it very useful in almost all cases. The only problem you can face by enabling this is that commenting out code blocks and saving the file results in lost imports, but it's very easy to restore them. Generally it helps keeping imports clean.

```json
{
  "editor.codeActionsOnSave": {
    "source.organizeImports": true
  }
}
```

Reply:
Yes. I also like to use the `recommendations` field.
And we'll add this when using JSON schemas:
```json
{
  "yaml.schemas": {
    "./packages/config-schema/schema.json": "nhost/config.yaml"
  }
}
```

Reply:
anyway I added the `.vscode` directory